src/librustc_mir/borrow_check/nll/type_check/liveness/trace.rs

```diff
@@ -37,6 +37,7 @@ pub(super) fn trace(
     flow_inits: &mut FlowAtLocation<'tcx, MaybeInitializedPlaces<'_, 'tcx>>,
     move_data: &MoveData<'tcx>,
     live_locals: Vec<Local>,
+    drop_used: Vec<(Local, Location)>,
```

Comment:
I found this pretty confusing. I finally realized that this is empty when not in polonius mode, but that was pretty non-obvious. Maybe we can rename this to like `polonius_drop_used_facts` and make it an `Option`?